rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/config/ArchiveIdentificationConfigLoadingRuleProvider.java

```diff
@@ -36,9 +39,15 @@
         return Initialization.class;
     }
 
+    // @Inject private StaticArchiveIdentifier identifier;
```

Comment:
Same as in other instances - to instantiate the right implementation of the service and inject it without boilerplate code like getInstance() method.